public/pages/VisualCreatePolicy/containers/CreateState/__snapshots__/CreateState.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<CreateState /> spec renders the component 1`] = `null`;
```

Comment:
Component not being rendered in snapshot for test.

Reply:
Whoops, fixed.